packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -233,63 +252,113 @@ function restLevelPackage(model: RLCModel) {
-    packageInfo.devDependencies["@types/mocha"] = "^10.0.0";
-    packageInfo.devDependencies["cross-env"] = "^7.0.2";
-    packageInfo.devDependencies["@types/chai"] = "^4.2.8";
-    packageInfo.devDependencies["chai"] = "^4.2.0";
-    packageInfo.devDependencies["cross-env"] = "^7.0.2";
-    packageInfo.devDependencies["karma-chrome-launcher"] = "^3.0.0";
-    packageInfo.devDependencies["karma-coverage"] = "^2.0.0";
-    packageInfo.devDependencies["karma-env-preprocessor"] = "^0.1.1";
-    packageInfo.devDependencies["karma-firefox-launcher"] = "^2.1.2";
-    packageInfo.devDependencies["karma-junit-reporter"] = "^2.0.1";
-    packageInfo.devDependencies["karma-mocha-reporter"] = "^2.2.5";
-    packageInfo.devDependencies["karma-mocha"] = "^2.0.1";
-    packageInfo.devDependencies["karma-source-map-support"] = "~1.4.0";
-    packageInfo.devDependencies["karma-sourcemap-loader"] = "^0.4.0";
-    packageInfo.devDependencies["karma"] = "^6.2.0";
+    if (model.options?.moduleKind === "cjs") {
+      packageInfo.module = `./dist-esm/src/index.js`;
+      packageInfo.devDependencies["mocha"] = "^10.0.0";
+      packageInfo.devDependencies["esm"] = "^3.2.18";
+      packageInfo.devDependencies["@types/mocha"] = "^10.0.0";
+      packageInfo.devDependencies["cross-env"] = "^7.0.2";
+      packageInfo.devDependencies["@types/chai"] = "^4.2.8";
+      packageInfo.devDependencies["chai"] = "^4.2.0";
+      packageInfo.devDependencies["cross-env"] = "^7.0.2";
+      packageInfo.devDependencies["karma-chrome-launcher"] = "^3.0.0";
+      packageInfo.devDependencies["karma-coverage"] = "^2.0.0";
+      packageInfo.devDependencies["karma-env-preprocessor"] = "^0.1.1";
+      packageInfo.devDependencies["karma-firefox-launcher"] = "^2.1.2";
+      packageInfo.devDependencies["karma-junit-reporter"] = "^2.0.1";
+      packageInfo.devDependencies["karma-mocha-reporter"] = "^2.2.5";
+      packageInfo.devDependencies["karma-mocha"] = "^2.0.1";
+      packageInfo.devDependencies["karma-source-map-support"] = "~1.4.0";
+      packageInfo.devDependencies["karma-sourcemap-loader"] = "^0.4.0";
+      packageInfo.devDependencies["karma"] = "^6.2.0";
+      packageInfo.scripts["build:browser"] =
+        "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1";
+      packageInfo.scripts["build:node"] =
+        "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1";
+      packageInfo.scripts["build:test"] = "tsc -p . && rollup -c 2>&1";
+    } else if (model.options?.moduleKind === "esm") {
+      packageInfo.devDependencies["@vitest/browser"] = "^1.3.1";
+      packageInfo.devDependencies["@vitest/coverage-istanbul"] = "^1.3.1";
+      packageInfo.devDependencies["playwright"] = "^1.41.2";
+      packageInfo.devDependencies["vitest"] = "^1.3.1";
+    }
     packageInfo.devDependencies["c8"] = "^8.0.0";
     packageInfo.devDependencies["source-map-support"] = "^0.5.9";
     packageInfo.devDependencies["ts-node"] = "^10.0.0";
```

Comment:
We should probably fix this to use `tsx` and that work versus `ts-node`

Reply:
We'll have this as a follow up effort